src/server.h

```diff
@@ -1899,7 +1900,8 @@ struct redisServer {
     int cluster_slave_no_failover;  /* Prevent slave from starting a failover
                                        if the master is in failure state. */
     char *cluster_announce_ip;  /* IP address to announce on cluster bus. */
-    char *cluster_announce_hostname;  /* hostname to announce on cluster bus. */
+    char *cluster_announce_hostname;  /* IP address to announce on cluster bus. */
```

Comment:
```suggestion
    char *cluster_announce_hostname;  /* hostname to announce on cluster bus. */
```

Hostnames don't need to be IP addresses, they can be FQDN as well.

Reply:
Addressed